src/app/(auth)/sign-in/page.tsx

```
@@ -0,0 +1,11 @@
+import { Metadata } from "next";
+import SignInView from "@/modules/auth/ui/views/sign-in-view";
+
+export const metadata: Metadata = {
+  title: "Sign In",
+};
+
+type Props = {};
+export default function SignInPage({}: Props) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)